From 8dbad6a4b4aabe757ee0dc1ad440ff8beb7a513c Mon Sep 17 00:00:00 2001
From: Mingliang Liu <liuml07@apache.org>
Date: Tue, 21 Feb 2017 17:30:39 -0800
Subject: [PATCH 2303/2748] HADOOP-14102. Relax error message assertion in S3A
 test ITestS3AEncryptionSSEC. Contributed by
 Mingliang Liu

(cherry picked from commit 0013090fb4340eadf147054e65a73de20a62c1c1)
(cherry picked from commit db955f0844ee2b961028fb2220b250e35d83e964)

Change-Id: I86a3e66155e62b7a2e76fed12924eceb4e6bc37d
---
 .../hadoop/fs/s3a/ITestS3AEncryptionSSEC.java      |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AEncryptionSSEC.java b/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AEncryptionSSEC.java
index 032ba0d..88eb426 100644
--- a/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AEncryptionSSEC.java
+++ b/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AEncryptionSSEC.java
@@ -65,7 +65,7 @@ public void testCreateFileAndReadWithDifferentEncryptionKey() throws
     Exception {
     expectedException.expect(java.nio.file.AccessDeniedException.class);
     expectedException.expectMessage(StringContains
-        .containsString("Forbidden (Service: Amazon S3; Status Code: 403;"));
+        .containsString("Service: Amazon S3; Status Code: 403;"));
 
     Path path = null;
     try {
-- 
1.7.9.5

